wrapper-jvm/src/main/java/eu/cloudnetservice/wrapper/transform/bukkit/FAWEConfigTransformer.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019-2024 CloudNetService team & contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package eu.cloudnetservice.wrapper.transform.bukkit;
+
+import eu.cloudnetservice.wrapper.transform.ClassTransformer;
+import java.lang.classfile.ClassTransform;
+import java.lang.classfile.CodeTransform;
+import lombok.NonNull;
+
+public class FAWEConfigTransformer implements ClassTransformer {
```

Comment:
```suggestion
@ApiStatus.Internal
public final class FAWEConfigTransformer implements ClassTransformer {
```
And I would really appreciate if you could add a small javadoc comment to all classes describing what the transformer is doing.